docs/sample_config.yaml

```diff
@@ -1594,6 +1594,15 @@ room_prejoin_state:
 #
 #track_appservice_user_ips: true
 
+# Set to establish a maximum backoff (in seconds) between HS -> AS connection attempts.
+# Upon failing to push appservice events, the homeserver will reattempt connection to the
+# application service after a delay. The delay increases with subsequent retries.
+# This value sets an upper limit on that delay.
+#
+# Regardless of this setting, the delay will never be longer than 512 seconds (about 8.5 minutes).
+#
+#appservice_max_backoff_s: 60
```

Comment:
Can this take a duration please? I.e. `appservice_max_backoff: 60s` instead. When reading the config you can use `self.parse_duration(...)`.